docs/source/development_release.rst

```diff
@@ -0,0 +1,80 @@
+.. _nbconvert_release:
+
+Making an ``nbconvert`` release
```

Comment:
Have you checked a built copy of this? I can't remember if backticks in headings work.